docs/DataModel.md

```diff
@@ -119,6 +119,65 @@ CREATE TABLE access_token (
 );
 ```
 
-## To Do
+## referer_tracker
+
+Keeps track of the mapping between the referer origin and the SSO state. When an external website calls Personal
+Data Server SSO API, this call is redirected to the authorization endpoint provided by the SSO provider. When making
+an authorization request, a SSO state is generated by Personal Data Server and sent to the SSO provider. At this
```

Comment:
"SSO state" isn't completely clear - is this the field which is used as the anti-forgery token? Some docs seem to refer to this as "anti-forgery state token" - could you refer to some external docs, and also to our own "Workflow", to explain where this state/token goes and perhaps name it something like "token" or "state token" or so? Thanks